src/Mutator/MutatorResolver.php

```diff
@@ -132,14 +132,14 @@ private static function resolveSettings($settings, array $globalSettings)
             return false;
         }
 
-        if (count($globalSettings) === 0) {
-            return (array) $settings;
-        }
-
         if ($settings === true) {
             return $globalSettings;
         }
 
+        if (count($globalSettings) === 0) {
```

Comment:
for me `$globalSettings === []` looks better. But it's up to you.

Reply:
```suggestion
        if ($globalSettings === []) {
```
It is certainly is. Missed this.